fe/src/main/java/org/apache/doris/alter/MaterializedViewHandler.java

```diff
@@ -507,6 +608,12 @@ private long checkAndGetBaseIndex(String baseIndexName, OlapTable olapTable) thr
         return baseIndexId;
     }
 
+    public void processBatchDropRollup (List<AlterClause> dropRollupClauses, Database db, OlapTable olapTable) throws DdlException {
+        for (AlterClause dropRollupClause : dropRollupClauses) {
+            processDropRollup((DropRollupClause)dropRollupClause, db, olapTable);
```

Comment:
"batch drop rollup" operation should still be atomic when writing meta log.
So you need to modify the `processDropRollup()`:

1. First, traverse all drop clauses to validate them, such as whether to drop the base index, does the rollup index need to be dropped exist.

2. Do the real drop operation. And all-or-non rollups should be dropped.

3. Write a edit log named `BatchDropRollupOperationLog` to persist the meta.